styleguide/source/assets/scss/01-atoms/_display-switch.scss

```diff
@@ -0,0 +1,22 @@
+.ama__display-switch {
+  display: inline-block;
+  border: solid 1px $purple;
+  border-radius: 5px;
+  padding: 1px 5px;
```

Comment:
should we use the padding mixin here?

Reply:
I avoided using the gutter mixin here since it's broken and I'd rather fix it in a separate branch, and also I can't remember if it accepts two values like this without having to create a new variable (that we probably wouldn't use again). I think it's okay to not use the mixin every time when there's good reason not to.